internal/randutil/randutil_test.go

```diff
@@ -0,0 +1,16 @@
+package randutil
```

Comment:
Can you add licensing to this file, `randutil.go`, and `uuid_test.go`? The license can start at 2022 for all since no license existed before.

Reply:
Ah, thank you for the reminder, I always forget that 🤦 . Added!